### PR TITLE
Add external configuration to enable RHOBS access

### PR DESCRIPTION
**Pull request by janboll**
Change prometheus reference to allow external prometheus configuration.

Required for: https://issues.redhat.com/browse/APPSRE-8197

### CODE REVIEW

File: graphql-schemas/schema.yml [hunk 1]
```diff
@@ -3663,9 +3663,25 @@ confs:
   - { name: labels, type: json }
   - { name: name, type: string, isRequired: true }
   - { name: app, type: App_v1, isRequired: true }
-  - { name: namespaces, type: Namespace_v1, isList: true, isRequired: true }
+  - { name: prometheusAccess, type: SLOPrometheusAccess_v1, isList: True, isRequired: true }
   - { name: slos, type: SLODocumentSLO_v1, isList: true }
 
+- name: SLOPrometheusAccess_v1
+  datafile: /app-sre/slo-document-1.yml
```
Comment (geoberle) [inline graphql-schemas/schema.yml:3670]: the datafile declaration should be used only on GQL types that are top-level to that datafile.

File: graphql-schemas/schema.yml [hunk 2]
```diff
@@ -3663,9 +3663,22 @@ confs:
...
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: namespace, type: Namespace_v1, isRequired: true}
+  - { name: prometheusAccess, type: SLOExternalPrometheusAccess_v1 }
+
+- name: SLOExternalPrometheusAccess_v1
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: url, type: string, isRequired: true }
```
Comment (hemslo) [inline graphql-schemas/schema.yml:3680]: `url` is required here but not in https://github.com/app-sre/qontract-schemas/pull/501/files#diff-25e1efebeeb9b86f33475e0ebbf223d3f568f35a72e4c6915e38e812f529a65eR33

Comment (janboll) [reply graphql-schemas/schema.yml:3680]: great spot, fixed!

**Review by hemslo — APPROVED** [review]
LGTM!

**Review by geoberle — APPROVED** [review]

**Pull request merged**